Packs/Maltiverse/README.md

```diff
@@ -0,0 +1,248 @@
+## Overview
+---
+
+Analyze suspicious hashes, URLs, domains and IP addresses
+This integration was integrated and tested with version xx of Maltiverse
+## Maltiverse Playbook
+---
+
+## Use Cases
+---
+
+## Configure Maltiverse on Demisto
+---
+
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
+2. Search for Maltiverse.
+3. Click __Add instance__ to create and configure a new integration instance.
+    * __Name__: a textual name for the integration instance.
+    * __API Key__
+    * __Trust any certificate (not secure)__
+    * __Use system proxy settings__
+4. Click __Test__ to validate the URLs, token, and connection.
+## Fetched Incidents Data
+---
+
+## Commands
+---
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+1. ip
+2. domain
+3. url
+4. file
+### 1. ip
+---
+Checks the reputation of an IP address
+##### Required Permissions
+**FILL IN REQUIRED PERMISSIONS HERE**
```

Comment:
If the API doesn't require specific permissions at all, we can go ahead and delete this.